frontend-shared/docs/frontend-shared.md

```diff
@@ -0,0 +1,71 @@
+# Frontend Shared Package
+
+
+The `frontend-shared` package is intended to serve as a common library for shared utilities, components and styles common to multiple Hypothesis front-end applications.
+
+The `frontend-shared` package source is located in `./frontend-shared` in this repository.
+
+Note: The long-term goal is to move this package into its own repository. There are some short-term advantages to the current setup: it piggy-backs on existing tooling and gives developers the benefit of working with just one repository when making changes (which primarily benefit the `client` in the first place).
+
+## How it works
+
+Frontend-shared works by creating a symlink via `$ yarn link` between the working dir
+`./frontend-shared/` and the installed dir `node_modules/@hypothesis/frontend-shared/`.
+
+## Client setup
+
+To set up `frontend-shared` in the client, run
+
+```shell
+$ make dev
+```
+
+as normal for development in the `client` application.
+
+
+This will trigger the `frontend-shared` build which:
+- Builds transpiled JSX into the `frontend-shared/lib` directory
+- Creates a linked package in the `node_modules/@hypothesis` folder (this is linked to `frontend-shared/lib`)
+
+When `$ make dev` is running, file changes will automatically trigger a rebuild. To rebuild manually, run
+
+
+```shell
+$ gulp setup-frontend-shared
+```
+
+
+## Usage in other repositories
+
+If you have modified the `frontend-shared` package locally and would like to preview changes in another Hypothesis applications that imports the package --that is, if you want to test uncommitted changes in `frontend-shared` in another local repository before publishing changes then follow these steps:
+
+1. In this repository's directory
+```shell
+    $ gulp setup-frontend-shared
+```
+
+1. In the target repository
+```shell
+    $ yarn link @hypothesis/frontend-shared
+```
+> Note: the package will need to be rebuilt after any changes are made to it before those changes can be seen in the consuming repository. This can be done by performing **either** of the following steps
+
+  - Running `$ make dev` in this repository, which will automatically re-build on any file changes.
```

Comment:
```suggestion
  - Run `$ make dev` in this repository, which will **automatically** re-build on any file changes, or
```

Reply:
In this variant, I'm suggesting the emphasis of the bit of this option that is most relevant: this is the option to handle things automatically (same thing with _manually_ in following bullet).